src/charm.py

```diff
@@ -149,19 +150,19 @@ def _on_install(self, _):
         service_enable(SRS_ENB_SERVICE)
         self._stored.installed = True
 
-    def _on_start(self, _):
+    def _on_start(self, _: StartEvent):
```

Comment:
We might want to add docstrings in this file as we're doing in `src/utils.py`, I don't know if you plan to do it in another PR.

Reply:
Done